scripts/extensions/sams/src/components/assets/assetPreviewPanel.tsx

```diff
@@ -83,6 +85,29 @@ export class AssetPreviewPanelComponent extends React.PureComponent<IProps> {
 
     render() {
         const {gettext} = superdeskApi.localization;
+        const actions: Array<IAssetCallback> =
+            [{
+                action: ASSET_ACTIONS.EDIT,
+                onSelect: this.onEditAsset,
+            },
+            {
+                action: ASSET_ACTIONS.DOWNLOAD,
+                onSelect: this.onDownloadSingleAssetCompressedBinary,
+            },
+            {
+                action: ASSET_ACTIONS.DELETE,
+                onSelect: this.onDeleteAsset,
+            },
+            ];
+
+        if (superdeskApi.privileges.hasPrivilege('sams_manage_assets')) {
+            actions!.push({
```

Comment:
The `!` should not be required here, as `actions` is initialised with a value directly before this statement